.clang-tidy

```diff
@@ -1,3 +1,3 @@
-Checks: '-*,kokkos-*,modernize-use-using,modernize-use-nullptr,cppcoreguidelines-pro-type-cstyle-cast'
+Checks: '-*,kokkos-*,modernize-type-traits,modernize-use-using,modernize-use-nullptr,cppcoreguidelines-pro-type-cstyle-cast'
 FormatStyle: file
-HeaderFilterRegex: '.*/*.hpp'
```

Comment:
This regex doesn't make much sense. We basically only check that the file ends in `hpp` and contains an arbitrary number of `/` and then an arbitrary character before that.

Reply:
I wonder who added it 😉

Reply:
I was young and naive. 🙂